crates/pathfinder/src/state.rs

```diff
@@ -217,39 +467,44 @@ async fn update<T: Transport>(
 
     // Download additional block information from sequencer. Use a custom timeout with retry strategy
     // to work-around the sequencer's poor performance (spurious lack of response) for early blocks.
```

Comment:
So I am not going to touch this part in the PR, but it sounds like it's something which should have it's own stage as planned and it can take all of the time in the world to retry these.